### PR TITLE
Fix Bugs in PE-D

### LINKED ISSUE #157: UD: Use numberings instead of indents for tables of content

### LINKED ISSUE #173: [PE-D][Tester B] Bug for edit grade test
Steps to reproduce
1. On the Students tab
2. editgradedtest 1 ra1/90 ra2/80 mt/85 f/88 pe/95

Expected
GUI to remain on Students tab so that I can view the newly edited marks

Actual
GUI switches to “Assignments” tab

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: Carlintyj/ped#3

### LINKED ISSUE #184: [PE-D][Tester C] User Guide Error
4 broken hyperlinks

- Broken exit hyperlink
- Broken archiving data files hyperlink
- Broken updating a task progress hyperlink
- Broken updating a session remark hyperlink

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: Jonyxzx/ped#1

### LINKED ISSUE #197: [PE-D][Tester D] editgradedtest ux bug
editgradedtest brings the user to the assignments page, however the gradedtests are only viewable from the students page, does not makes sense to redirect users to assignments page, can consider redirecting to the students page instead for better ux.

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: samuelmui8/ped#9

### LINKED ISSUE #198: [PE-D][Tester E] "editgrade" command in UG is inconsistent with app
For the "editgrade" command, I tried only editing certain fields such as grades for ra1 or ra2 only and it worked but it does not reflect in the UG that these fields are optional. Could consider adding that it through "[]" as mentioned in the beginning of the UG. 

The command at the header ("editgradedtest") does not match the one that is given in the command format ("editgrade")

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: zannloo/ped#6

### LINKED ISSUE #217: UG: No emoji
image

### PR DESCRIPTION
**Pull request by raydenlim**
Fix GradedTest tab
Fix helpMessage.png Image in UG

### CODE REVIEW

**Comment by raydenlim** [comment]
Fix issue of no ❌ for `Delete a Student`

File: docs/UserGuide.md [hunk 1]
```diff
@@ -102,7 +157,7 @@ The F.A.K.E.J.A.R.V.I.S. User Guide employs a variety of visual cues to enhance
 | Convention                                                                          | Description                                                     |
 |-------------------------------------------------------------------------------------|-----------------------------------------------------------------|
 | **<div markdown="span" class="alert alert-info"> :information_source: Note </div>** | Provides information of special interest or importance.         |
-| **<div markdown="span" class="alert alert-warning"> :bangbang: Warning </div>**     | Alerts to potentially irreversible actions with data loss risk. |
+| **<div markdown="span" class="alert alert-warning"> :bangbang: Warning </div>**     | Alerts to potentially irreversible actions with data loss risk. |z
```
Comment (Respirayson) [inline docs/UserGuide.md]: perhaps a typo here

**Review by Respirayson — APPROVED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (raydenlim) [reply docs/UserGuide.md]: yikes

**Comment by raydenlim** [comment]
Updated TOC format, increased indentation
Fix inconsistencies in UG

**Pull request merged**